app/javascript/components/explore/DifferentialExpressionPanel.jsx

```diff
@@ -60,24 +62,25 @@ export default function DifferentialExpressionPanel({
         setDeGenes={setDeGenes}
         deGroup={deGroup}
         setDeGroup={setDeGroup}
-        setDeFileUrl={setDeFileUrl}
         countsByLabel={countsByLabel}
         deObjects={deObjects}
+        setDeFilePath={setDeFilePath}
       />
 
       {deGenes &&
       <>
         15 most DE genes
-        <span className="pull-right">
-          {/* <a href={deFileUrl}
-            target="_blank"
-            data-analytics-name="differential-expression-download"
-            data-toggle="tooltip"
-            data-original-title="Download all DE genes data for this group"
-          >
-            <FontAwesomeIcon icon={faDownload}/></a> */}
-          <DifferentialExpressionModal />
-        </span>
+        <a className="de-download-button"
+          onClick={async () => {await downloadBucketFile(bucketId, deFilePath)}}
```

Comment:
Have you tried this without the async / await wrapper?  I.e.:

```suggestion
          onClick={downloadBucketFile(bucketId, deFilePath)}
```

Even though `downloadBucketFile` is an async function, I don't see off-hand why the onClick handler would need to be async too.  If it works without that, this would could slightly reduce visual noise / improve maintainability.

Reply:
It's most proper/standard to `await` (and therefore wrap that function with `async` or else you will get an `Uncaught SyntaxError: await is only valid in async function`) when calling an `async` function. I would argue that then makes this more maintainable since it makes it more obvious that `downloadBucketFile()` is an `async` function and is the most standard practice.